### ISSUE TITLE
Resume Skips to Next Track

### ISSUE THREAD

**Issue by zachmoody** [gillesdemey/Cumulus#81]
Wondering if anyone else has seen this before. When I pause a track for a while and come back it'll resume where I left off for ~30s then skip to the next track in my stream. It seems to have started around `0.8.0` and is still doing it on `0.9.0`.

**Comment by seppestas** [comment]
Hey @zachmoody, could you specify in what environment (operating system, window manager, ...) you have this issue?

**Comment by zachmoody** [comment]
Hey, sorry, MacOS 10.12.2

**Comment by chrisjansky** [comment]
Confirming the same issue on macOS 10.11.6, Cumulus 0.10.0.

**Comment by andersryanc** [comment]
I think I'm experiencing a related issue... I just downloaded a fresh copy today (0.10.1) and when I first opened the app and hit play, it immediately skipped through the entire list (and appeared to mark them all as listened). No sound was played... and I got a whole bunch of notifications of each song playing one after another... I tried to hit pause, but it just kept skipping through tracks

<img width="363" alt="screen shot 2017-06-09 at 11 06 33 am" src="https://user-images.githubusercontent.com/689046/26988338-c8552b00-4d03-11e7-9ef6-45e86f6e6f2b.png">
<img width="323" alt="screen shot 2017-06-09 at 11 07 02 am" src="https://user-images.githubusercontent.com/689046/26988337-c851da72-4d03-11e7-96b1-979e3e7cd816.png">

**Comment by zachmoody** [comment]
@andersryanc think that behavior is related to #89.

**Comment by andersryanc** [comment]
Yep... after downloading the source and investigating... I can see I'm getting the same rate limit error when I load up the stream URL in a browser:

<img width="856" alt="screen shot 2017-06-09 at 11 26 16 am" src="https://user-images.githubusercontent.com/689046/26989212-16f9d85c-4d07-11e7-8be9-d84f58c35906.png">

`{"errors":[{"meta":{"rate_limit":{"bucket":"by-client","max_nr_of_requests":15000,"time_window":"PT24H","name":"plays"},"remaining_requests":0,"reset_time":"2017/06/09 20:41:59 +0000"}}]}`

Thanks

**Comment by Hum4n01d** [comment]
@gillesdemey @MichielDeMey Close this :)

**Comment by gillesdemey** [comment]
It's unclear if the original issue is related, we weren't running into rate limits back then.

**Comment by Hum4n01d** [comment]
Oh my bad, I was talking about the other issue people were talking about in this thread. Sorry about that

**Comment by oleschaper** [comment]
Same here, v0.10.1, MacOSX 10.14.3
